cmd/main.go

```diff
@@ -1,30 +1,34 @@
 package main
 
 import (
+	"flag"
 	"fmt"
-	"io/ioutil"
-	"log"
-	"net/http"
 	"os"
+	"weather"
 )
 
+type Response struct {
+	OneWord string  `json:"oneword"`
```

Comment:
Maybe `Summary`?

Reply:
I don't think we ever marshal these structs as JSON, do we? So they don't need tags for the time being.